kdd-2024/ibis-tutorial-KDD2024.md

```diff
@@ -0,0 +1,273 @@
+# Descriptive Title 
+
+We need a title here, in the past we had this options
+
+- Scipy and PyCon 2024: Intro to Ibis: blazing fast analytics with DuckDB, Polars, Snowflake, and more,
```

Comment:
How about: "Ibis: portable and blazing fast analytics with DuckDB, Polars, Snowflake, and more..."

Reply:
I'm worried that we won't capture the DS and ML crowd by using the word "analytics" but idk.